include/z64audio.h

```diff
@@ -86,12 +93,12 @@ typedef enum {
 } AudioCacheType;
 
 typedef enum {
-    /* 0 */ LOAD_STATUS_0,
-    /* 1 */ LOAD_STATUS_1,
-    /* 2 */ LOAD_STATUS_2, // Samples/Seqplayer
-    /* 3 */ LOAD_STATUS_3, // Sequences
-    /* 4 */ LOAD_STATUS_4, // SoundFonts
-    /* 5 */ LOAD_STATUS_5 // Permanent
+    /* 0 */ LOAD_STATUS_NOT_LOADED,
```

Comment:
```suggestion
    /* 0 */ LOAD_STATUS_UNLOADED,
```

Not loaded is redundant. I'm less confident in this change than the permanent one.

Maybe closed instead of unloaded? Either way, unloaded is shorter than not loaded.

Reply:
`UNLOADED` makes it seem like it was in a previously `LOADED` state before being unloaded which doesn't need to be true with this status. "closed" also doesn't really seem clear to me. I don't think `NOT_LOADED` is redundant, It specifies that the status is that it hasn't been loaded yet

Reply:
I don't think unloaded carries an implication of having been loaded in the past, but I don't feel strongly on this point.